### PR TITLE
Project structure with basic code rules

### PR DESCRIPTION
**Pull request by freddy-perez-jalafund**
This is the base structure for web app with react

### CODE REVIEW

**Review by DaleneyRuben — COMMENTED** [review]
It's looking all good @freddy-perez-jalafund 
Maybe we can add a hook to _**format**_  the code before a commit. So this way we won't care if the guys run prettier in their code, we'll get well formatted code when they decide to push it. [Husky](https://github.com/typicode/husky) is a library that can help us dealing with that. 

EDIT: I'm assuming we're going to use npm so please include the **package-lock.json** in this PR.

**Comment by martin-callejas-jalafund** [comment]
@fernando-maceda-fundacion husky bro

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
```
Comment (freddy-perez-jalafund) [inline .eslintrc:1]: Is only the basic for prettier. There is another task that will add corresponding hooks

**Comment by freddy-perez-jalafund** [comment]
> It's looking all good @freddy-perez-jalafund
> Maybe we can add a hook to _**format**_ the code before a commit. So this way we won't care if the guys run prettier in their code, we'll get well formatted code when they decide to push it. [Husky](https://github.com/typicode/husky) is a library that can help us dealing with that.
> 
> EDIT: I'm assuming we're going to use npm so please include the **package-lock.json** in this PR.

There is another task that will add corresponding hooks and updated for now only is a structure of project

**Review by DaleneyRuben — APPROVED** [review]
LGTM 👍🏻

**Review by jorge-lema-jalafund — APPROVED** [review]
approved!

**Pull request merged**